test/IRGen/raw_layout.swift

```diff
@@ -241,32 +287,106 @@ entry(%0 : $*Cell<T>):
 // CHECK: [[T_ADDR:%.*]] = getelementptr inbounds ptr, ptr %"CellThatMovesAsLike<T>", {{i64|i32}} 2
 // CHECK-NEXT: [[T:%.*]] = load ptr, ptr [[T_ADDR]]
 // CHECK: {{%.*}} = call ptr %InitializeWithTake(ptr {{.*}} %dest, ptr {{.*}} %src, ptr [[T]])
 
+//===----------------------------------------------------------------------===//
 // CellThatMovesAsLike<T> assignWithTake
+//===----------------------------------------------------------------------===//
 
 // CHECK-LABEL: define {{.*}} ptr @"$s10raw_layout19CellThatMovesAsLikeVwta"(ptr {{.*}} %dest, ptr {{.*}} %src, ptr %"CellThatMovesAsLike<T>")
 // CHECK: [[T_ADDR:%.*]] = getelementptr inbounds ptr, ptr %"CellThatMovesAsLike<T>", {{i64|i32}} 2
 // CHECK-NEXT: [[T:%.*]] = load ptr, ptr [[T_ADDR]]
 // CHECK: {{%.*}} = call ptr %AssignWithTake(ptr {{.*}} %dest, ptr {{.*}} %src, ptr [[T]])
 
+//===----------------------------------------------------------------------===//
 // ConcreteMoveAsLike initializeWithTake
+//===----------------------------------------------------------------------===//
 
 // CHECK-LABEL: define {{.*}} ptr @"$s10raw_layout18ConcreteMoveAsLikeVwtk"(ptr {{.*}} %dest, ptr {{.*}} %src, ptr %ConcreteMoveAsLike)
 // CHECK: [[DEST_CELL:%.*]] = getelementptr inbounds %T10raw_layout18ConcreteMoveAsLikeV, ptr %dest, i32 0, i32 0
 // CHECK: [[SRC_CELL:%.*]] = getelementptr inbounds %T10raw_layout18ConcreteMoveAsLikeV, ptr %src, i32 0, i32 0
 // CHECK: {{invoke void|invoke ptr|call ptr}} @{{.*}}(ptr [[DEST_CELL]], ptr [[SRC_CELL]])
 
+//===----------------------------------------------------------------------===//
 // ConcreteMoveAsLike assignWithTake
+//===----------------------------------------------------------------------===//
 
 // CHECK-LABEL: define {{.*}} ptr @"$s10raw_layout18ConcreteMoveAsLikeVwta"(ptr {{.*}} %dest, ptr {{.*}} %src, ptr %ConcreteMoveAsLike)
 // CHECK: [[DEST_CELL:%.*]] = getelementptr inbounds %T10raw_layout18ConcreteMoveAsLikeV, ptr %dest, i32 0, i32 0
 // CHECK: [[SRC_CELL:%.*]] = getelementptr inbounds %T10raw_layout18ConcreteMoveAsLikeV, ptr %src, i32 0, i32 0
 // CHECK: {{invoke void|invoke ptr|call ptr}} @{{.*}}(ptr [[DEST_CELL]], ptr [[SRC_CELL]])
+
+//===----------------------------------------------------------------------===//
+// SmallVectorOf2MovesAsLike<T> initializeWithTake
+//===----------------------------------------------------------------------===//
+
+// CHECK-LABEL: define {{.*}} ptr @"$s10raw_layout25SmallVectorOf2MovesAsLikeVwtk"(ptr {{.*}} %dest, ptr {{.*}} %src, ptr %"SmallVectorOf2MovesAsLike<T>")
+// CHECK: [[T_ADDR:%.*]] = getelementptr inbounds ptr, ptr %"SmallVectorOf2MovesAsLike<T>", {{i64|i32}} 2
+// CHECK-NEXT: [[T:%.*]] = load ptr, ptr [[T_ADDR]]
+// CHECK: [[STRIDE_GEP:%.*]] = getelementptr inbounds %swift.vwtable, ptr {{%.*}}, i32 0, i32 9
+// CHECK-NEXT: [[STRIDE:%.*]] = load {{i64|i32}}, ptr [[STRIDE_GEP]]
+// CHECK: [[OFFSET_0:%.*]] = mul i64 0, [[STRIDE]]
```

Comment:
@Azoy Seems like the `mul` index operand types also need to be `{{i64|i32}}`
```
/home/build-user/build/buildbot_linux/wasmstdlib-linux-x86_64/test-wasi-wasm32/IRGen/Output/raw_layout.swift.tmp/raw_layout.sil:339:11: error: CHECK: expected string not found in input
// CHECK: [[OFFSET_0:%.*]] = mul i64 0, [[STRIDE]]
          ^
<stdin>:3260:28: note: scanning from here
 %stride = load i32, ptr %2, align 4, !invariant.load !5
                           ^
<stdin>:3260:28: note: with "STRIDE" equal to "%stride"
 %stride = load i32, ptr %2, align 4, !invariant.load !5
                           ^
<stdin>:3261:2: note: possible intended match here
 %3 = mul i32 0, %stride
```
https://ci.swift.org/job/oss-swift-pr-test-crosscompile-wasm-ubuntu-20_04/1253/console